openstack/ironic/templates/redfish-certrobot-cronjob.yaml

```
@@ -28,6 +28,7 @@ metadata:
   name: {{ .Release.Name }}-redfish-certrobot-0
 spec:
   schedule: {{ .Values.cert_robot.schedule | quote }}
+  timeZone:  {{ .Values.cert_robot.timezone | default "Etc/UTC" | quote }}
```

The `default` is redundant because you already have the same default in `values.yaml`:
```suggestion
  timeZone:  {{ .Values.cert_robot.timezone | quote }}
```